### DISCUSSION TITLE
Trojan warning getting host 7z

### DISCUSSION THREAD

**Discussion by PsychoFox11** [DarkMatterCore/nxdumptool#175] [Q&A]
Trying to get the host 7z, but Windows Defender identifies a Trojan in it. Don't remember having this issue before. I have had to whitelist another app (the analogue pocket updater 'pupdate') from github after the dev confirmed it was clean. Is this a known thing and safe to actually grab, or did something slip in?

**Comment by Whovian9369** [comment]
It's a frustrating known issue, unfortunately. There's no malware in it, we promise.

To elaborate on that, we believe that since it's a repackaged Python script, antivirus software are automatically flagging it for that fact.
The original Python script that generates the `nxdt_host.exe` is available at https://github.com/DarkMatterCore/nxdumptool/blob/rewrite/host/
You can also use the provided `bat` script to repackage the script yourself, if you'd like to test with that. (You may need to install the Python prerequisites yourself using `pip` before running the script.)
